browserslist

```diff
@@ -2,3 +2,4 @@ last 2 chrome versions
 last 2 edge versions
 last 2 firefox versions
 last 2 safari versions
+maintained node versions
```

Comment:
I've also added this so we don't accidentally push js syntax that might break in jest tests ran with Node LTS versions.